src/main/java/genji/Storage.java

```diff
@@ -0,0 +1,81 @@
+package genji;
+
+import genji.task.Task;
+import genji.task.TaskList;
+import genji.task.ToDo;
+import genji.task.Deadline;
+import genji.task.Event;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Storage {
+   private String filePath;
+
+   public Storage(String filePath) {
+       this.filePath = filePath;
+   }
+
+    public TaskList loadList() {
+       TaskList list = new TaskList();
+       try {
+            File f = new File(filePath);
+            Scanner s = new Scanner(f);
+            if (!s.hasNextLine()) {
+                System.out.println("No task in list");
+            }
+            while (s.hasNext()) {
+                String input = s.nextLine();
+                if (input.startsWith("T")) {
+                    list.add(new ToDo(input.substring(8)));
+                    if (input.substring(4).startsWith("1")) {
+                        list.get(list.size()- 1).mark();
+                    }
+                } else if (input.startsWith("D")) {
+                    String name = input.substring(8, input.lastIndexOf(" |"));
```

Comment:
Possibly a small suggestion to add lines between code to declutter it, making it more readable.